src/main/kotlin/racingcar/domain/LapReport.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar.domain
+
+class LapReport(
+    val lapReport: List<Pair<CarName, Location>>
+) {
+    fun winner(): List<CarName> {
+        val topLocation = topLocation()
+        return lapReport.filter { it.second == topLocation }
+            .map { it.first }
```

Comment:
```suggestion
        return lapReport
            .asSequence()
            .filter { it.second == topLocation }
            .map { it.first }
            .toList()
```
처리 단계가 두 단계 이상인 경우에는 시퀀스를 활용하는 것을 권장해요.